spec/calendar.html

```diff
@@ -416,7 +416,18 @@ <h1>ToISOWeekOfYear ( _year_, _month_, _day_ )</h1>
       <emu-note>Beware that dates at the beginning of a year may be part of a week from the preceding year, and dates at the end of a year may be part of a week at the beginning of the next year, as the first week of any year is defined as the week that contains the first Thursday of the year.</emu-note>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-resolvenonlunisolarmonth" aoid="ResolveISOMonth">
+    <emu-clause id="sec-buildisomonthcode" aoid="BuildISOMonthCode">
+      <h1>BuildISOMonthCode ( _month_ )</h1>
+      <p>
+        The abstract operation BuildISOMonthCode returns the corresponding string month code, given the numeric ISO calendar month _month_.
```

Comment:
is “numeric iso calendar month” defined somewhere explicitly so that this description functions as a type assertion?

Reply:
I can add that it's a number between 1 and 12, or write an explicit assertion algorithm step.

Reply:
As long as the assertion is explicit in either prose or steps, I’m happy :-) preference beyond that is up to the editors.